Desafio_Snowman/use_cases_turismo.py

```diff
@@ -30,6 +28,26 @@ def haversine(lon1, lat1, lon2, lat2):  # def que aplica a formula de haversine
     r = 6371
     return c * r
 
+def token_required(f):
```

Comment:
I would suggest moving this function to the auth module.

Reply:
But wouldn't that generate circular dependency? in **use_case_turismo.py** is imported **auth.py** and, how the function of validate user exist in **use_case_turismo.py**, i  would need import her in **auth.py**.